sdk/storage/mgmt/src/main/java/com/azure/management/storage/implementation/BlobContainerImpl.java

```diff
@@ -58,15 +58,15 @@ public StorageManager manager() {
     public Mono<BlobContainer> createResourceAsync() {
         BlobContainersInner client = this.manager().inner().blobContainers();
         return client
-            .createAsync(this.resourceGroupName, this.accountName, this.containerName, cpublicAccess, cmetadata)
+            .createAsync(this.resourceGroupName, this.accountName, this.containerName, this.inner())
```

Comment:
You need to change the set part? e.g. `withPublicAccess`

Reply:
I'll take a look.

Reply:
Add parameter done.